tests/ard_reprocessed_l1s/db_index.sh

```diff
@@ -31,7 +36,9 @@ fi
 if [[ $HOSTNAME == *"LAPTOP-UOJEO8EI"* ]]; then
   echo "duncans laptop"
   echo "conda activate /home/duncan/bin/miniconda3/envs/odc2020"
-  ODCCONF="--config ${SCRIPT_DIR}/duncan_local.conf"
+  echo "sudo service postgresql start"
+  echo "This env does not have hd5, so do not do ARD processing"
+  ODCCONF="--config ${SCRIPT_DIR}/${USER}_local.conf"
```

Comment:
i noticed we removed the line which refers to the duncan_local.conf file (and that file is added to this review) thus for odcconf, we use the local user's conf  (ie. dsg547_local.conf or gy5636_local.conf) . 
Why is duncan_local.conf being added to the repository?

Reply:
This is so tests work locally, with a user name of duncan